### ISSUE TITLE
Project/documentation site is blank when viewed over HTTPS

### ISSUE THREAD

**Issue by csuhta** [NYTimes/backbone.stickit#272]
https://nytimes.github.io/backbone.stickit/ is rendering a blank screen for me in Safari 7.1.2:

![ss 2015-01-26 at 10 43 06 am](https://cloud.githubusercontent.com/assets/769083/5902401/327185ee-a548-11e4-830c-632625706c5a.png)

The JS error is: 

```text
https://documentup.com/nytimes/backbone.stickit?config=%7B%22twitter%22%3A%5B%22delambro%22%5D%2C%22google_analytics%22%3A%22UA-36460916-1%22%7D&callback=callback
Failed to load resource: the server responded with a status of 521 (Internal Server Error)
```

The non-HTTPS version works fine. 

The HTTPS version is the one Google links me to when I search for "backbone stickit"

**Comment by akre54** [comment]
Ah yep. [Documentup](https://github.com/jeromegn/DocumentUp), which is responsible for rendering the README, looks like it's having trouble in that area. Mind opening an issue at that repo and reporting back?

(as an aside, it looks like visiting http://documentup.com works, while https is showing 521 so it's likely site-wide.)

**Comment by csuhta** [comment]
I'm not familiar enough with your documentation config to open an issue there. (It feels like you're asking me to open something akin to a domain or IT-releated ticket for you.)

**Comment by akre54** [comment]
Nah, I just meant alerting them with an open issue. No big deal, I'll make one now.
